src/operator/src/insert.rs

```diff
@@ -238,8 +202,34 @@ impl Inserter {
             ..Default::default()
         });
 
-        let tasks = self
-            .group_requests_by_peer(requests.clone())
+        // Mirror requests for source table to flownode
+        match self.mirror_flow_node_requests(&requests).await {
```

Comment:
What about adding a metric here?

Reply:
1. Added mirror rows count metric,
2. We already have metric in the cache manager